### DISCUSSION TITLE
Contributor call: 27 May 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#4048] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: May 27 2023 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/may-27-2023/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- Waist to Underarm. #1360 
  - In the 2 October 2021 Contributor Call, did we already agree to make this an official measurement? #1353 If not, do we want to make this an official measurement?
  - Did we agree to use the name "Waist to Armhole"? (If not, my suggestion is to use "Waist to Underarm".)
- HPS to Shoulder.#2850 Do we want to add this as an official measurement?
- Are there any other official measurements we should be considering at this time, before v3?
- Social media and the lack thereof
- The v3 UI

Comment below to add things to the agenda

**Comment by BenJamesBen** [comment]
- Waist to Underarm. #1360 
  - In the 2 October 2021 Contributor Call, did we already agree to make this an official measurement? #1353
  - If not, do we want to make this an official measurement?
  - Did we agree to use the name "Waist to Armhole"? (If not, my suggestion is to use "Waist to Underarm".)
- HPS to Shoulder. #2850
  - Do we want to add this as an official measurement?
  - (If so, we would need to determine neckstimate values for this, for cisFemale neck 34 cm and cisMale neck 38 cm. and also the scaling correction ratio.)
- Are there any other official measurements we should be considering at this time, before/for v3?

**Comment by BenJamesBen** [comment]
Discussion about the new and updated designs in the PR stage. _(What is the status of each PR? For each, what needs to be done next, and how can we help shepherd them towards merge?)_
- Lily leggings. https://github.com/freesewing/freesewing/pull/3053
- Dona shirtdress. https://github.com/freesewing/freesewing/pull/3876
- Titan angled waist. https://github.com/freesewing/freesewing/pull/3493
- Senya circle shirt. https://github.com/freesewing/freesewing/pull/3762
- Skully plush. https://github.com/freesewing/freesewing/pull/3797
- Otis romper. https://github.com/freesewing/freesewing/pull/3773
- Shelly swimshirt. https://github.com/freesewing/freesewing/pull/4127, https://github.com/freesewing/freesewing/pull/4128

Quick update on other designs:
- Turtleneck. Stalled. Difficulties in changing concepts from a single garment to a parametric pattern.
- Magde messenger bag. Merged. Pattern designer has been busy but is working on writing the docs.
- Tortuga pirate shirt. Stalled. Temporarily neglected by pattern designer. 
- (Any others new designs under development that want to provide a quick update?)

Are there any other stalled PRs we should look at?
- Perhaps https://github.com/freesewing/freesewing/pull/2953 ?

**Comment by tangerineshark** [comment, marked as answer]
### Notes

General Discussion

- joost is sick :-(
- Thrunic is working on a unisuit pattern :-)
- It is discovered that @woutervdub , @eriese , and @tangerineshark all have very different definitions of an ideal hike.

Waist to Underarm. https://github.com/freesewing/freesewing/issues/1360
In the 2 October 2021 Contributor Call, did we already agree to make this an official measurement? https://github.com/freesewing/freesewing/issues/1353 If not, do we want to make this an official measurement?
Did we agree to use the name "Waist to Armhole"? (If not, my suggestion is to use "Waist to Underarm".)

- It was decided to change the name to "waist to armpit", as underarm is a bit confusing.
- Issue #1360 should be added to the roadmap (tagging @joostdecock for that?).
- Hurdle: ideally, we would also adjust existing blocks and patterns to use this, but that's up to the individual maintainers (although fortunately, most of them are @joostdecock ).

HPS to Shoulder.https://github.com/freesewing/freesewing/issues/2850 Do we want to add this as an official measurement?
Are there any other official measurements we should be considering at this time, before v3?

- This is a request from a FreeSewing superuser who makes awesome stuff, but always has to adjust his shoulder measurements on FS garments.
- However, "shoulder to shoulder" and "neck circumference" should be able to derive this, and HPS works a bit better for vertical measies, where having the HPS a little off is less likely to affect the outcome dramatically. This means that there's a bit of overlapping measies if this were implemented, and a higher likelihood of folks measuring their HPS-based measurements incorrectly.
- Side note: a cool option would be to standardize the way neck circumference and shoulder-to-shoulder create the HPS to shoulder, so you could check it.
- It is decided that we won't implement #2850 , but we will reach out to the user and look for alternative options that can resolve this challenge

Vertical trunk measurement

- This is a measurement that Thrunic would like to use, as it would help in the creation of a unisuit pattern
- However, it can be derived from other existing measurements.
- Therefore, we're gonna add it to the plugins as a calculated measurement instead.
- As a bonus, it's a great way to doublecheck that the measurements summing up to "vertical trunk" are accurate.

Social media and the lack thereof

- @tangerineshark tabled this conversation for next Contributor Call, and is adding it to that agenda now: 

The v3 UI

- Goal is to have @BenJamesBen 's long list of patterns added for v3 (what is @tangerineshark referencing here? I think a big list of patterns waiting to be published, which Ben has kindly pulled together), but @joostdecock is focusing on the website first (then will add these patterns while folks are testing the UI). That way, patterns can be implemented "the v3 way", although it's not a requirement.
- @eriese shares lots of love for the UI, along with a few things to improve, including:
  - there are lots of accordions, and some could be toggles
  - there's a lot of complexity, and ze sometimes has to go back and change the UI in certain cases
  - @joostdecock and @eriese serendipitously suggest that the control settings could be used to control the style of inputs
  - @eriese is refactoring inputs currently so that they use same components (@tangerineshark may have missed the crux of this, but an example was that all booleans will change at once)
  - the home icon is a little hard to find, and @joostdecock said he'll make it more obvious (this may be partly due to some menus currently disappearing in dark mode, although they are technically still there).
  - Currently, you lose your settings state if you leave the page, if you don't make an account. A warning for this would be good. Or you could toggle on a save button, but a warning would be easier to implement.

As always, anything confusing or inaccurate is the fault of @tangerineshark 😅 🧡 🦈

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
